sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1308,8 +1307,8 @@ public void testBuildSourceDisplayData() {
   }
 
   @Test
-  @Category(RunnableOnService.class)
-  @Ignore("[BEAM-436] DirectRunner RunnableOnService tempLocation configuration insufficient")
+  @Category(NeedsRunner.class)
```

Comment:
RunnableOnService is desired here. We want to check that runners preserve the display data.